extensions/integration-tests/src/main.ts

```diff
@@ -12,16 +12,21 @@ const TEST_SETUP_COMPLETED_TEXT: string = 'Test Setup Completed';
 const EXTENSION_LOADED_TEXT: string = 'Test Extension Loaded';
 const ALL_EXTENSION_LOADED_TEXT: string = 'All Extensions Loaded';
 
+const logPrefix: string = 'test:int:main';
+const debug: any = require('debug')(logPrefix);
+const trace: any = require('debug')(`${logPrefix}:trace`);
```

Comment:
I guess I don't really have a problem with this change despite being unrelated to this PR, but are you planning on coming back and doing this in more places? Seems kind of arbitrary if we're not planning on adopting this in a larger fashion.